core/src/com/unciv/logic/civilization/diplomacy/DiplomacyManager.kt

```diff
@@ -96,6 +96,9 @@ class DiplomacyManager() {
     /** Total of each turn Science during Research Agreement */
     var totalOfScienceDuringRA = 0
 
+    /** Flag set to true when the two civs have been friends in the past */
+    var everBeenFriends: Boolean = false
```

Comment:
If this is a flag, it should be set as a diplomacyFlag rather than a new field

Reply:
It is a flag, but it has no countdown. It should never expire.. how do you suggest to handle this?

Reply:
Set -1 for flags that don't expire, and countdown should just ignore them
I assume we'll be seeing more of these kind of flags in the future...